src/test/java/core/basesyntax/services/operations/FruitOperationStrategyAndFruitTransactionProcessorTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.services.operations;
+
+import static core.basesyntax.db.Storage.fruitStorage;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.exceptions.FruitStorageException;
+import core.basesyntax.services.FruitTransactionProcessor;
+import core.basesyntax.services.impl.FruitTransactionProcessorImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitOperationStrategyAndFruitTransactionProcessorTest {
```

Comment:
this class does not sound very "Single responsibility" to me 😅 
let's split them up